### PR TITLE
Updated batcher tests; added channel transfer with batcher test

### PR DESCRIPTION
**Pull request by zemtsov**
- Batcher tests updated with using TestSuite.
- New test for channel transfer with batcher added.
- TestSuite extended with execute tasks methods.

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -111,7 +111,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        INTEGRATION_TEST_SUITE: [ "general basic","swap","version-and-nonce","channel-transfer-only-tx","channel-transfer","channel-transfer-multi","batcher-only-tx"]
+        INTEGRATION_TEST_SUITE: [ "general basic","swap","version-and-nonce","channel-transfer-only-tx","channel-transfer","channel-transfer-multi","batcher-only-tx","channel-transfer-batcher-only-tx"]
```
Comment (pfi79) [inline .github/workflows/go.yml:114]: change the line like this
```
"general basic","swap","version-and-nonce","channel-transfer-only-tx","channel-transfer","batcher-only-tx channel-transfer-multi channel-transfer-batcher-only-tx"
```

**Comment by pfi79** [comment]
@AtapinDmitry need your review.

File: .github/workflows/go.yml (hunk 1 above)
Comment (zemtsov) [reply .github/workflows/go.yml]: Done

**Comment by AtapinDmitry** [comment]
Approved

**Review by pfi79 — APPROVED** [review]

**Pull request merged**